### ISSUE TITLE
Global setting: tempdir

### ISSUE THREAD

**Issue by CMargreitter** [MolecularAI/Icolos#7]
Would be great to be able to set the tempdir location in the global settings if required (e.g. to make use of some faster drive). Will involve a lot of updates in various steps, though.

**Comment by jharrymoore** [comment]
Sure, so as I understand it, it would be a prefix to the tmpdir to specify somewhere other than /tmp or /scratch/ as the default directory for the tmpfiles? I will add add that in time for the next release

**Comment by CMargreitter** [comment]
Exactly. Unfortunately, it seems there's no easy solution (such as exporting an env variable) that works in all cases. So it might be required to replace mkstemp etc. calls by a function that (1) tries to load the global variable and checks whether the path is valid and (2) default to the standard location if not.
